backend/dal/Migrations/20210122232754_v01.06.01.cs

```diff
@@ -0,0 +1,22 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+using Pims.Dal.Helpers.Migrations;
+
+namespace Pims.Dal.Migrations
+{
+    public partial class v010601 : SeedMigration
```

Comment:
Just be aware that @devinleighsmith has a migration with the same number in a recent PR